supplementary/alica_tracing/include/tracing/Trace.h

```diff
@@ -52,7 +52,7 @@ class Trace : public alica::IAlicaTrace
     //        {"wait.time", "waited 10 sec for agent 2"}});
     // BE CAREFUL about what you want to log,
     // These are supposed to be micro logs to be carried over the network.
-    void setLog(std::pair<std::string, std::string> logEntry) override;
+    void setLog(const std::pair<std::string, TraceValue>& logEntry) override;
```

Comment:
oos but I think we should also get rid of the pair syntax here and just take two parameters, at least in devel

Reply:
I think log actually allows multiple key-value pairs, should this be supported?
Maybe mark this as deprecated and add a new method?

Reply:
I wonder how it shows up in jaeger UI for multi log pairs.  Ideally for a single pair we shouldn't need to wrap in `{}`.  We can also support multi pairs if it seems useful.
Yeah I'd support deprecating this and adding new methods.  Later we can do the same for amr_tracing

Reply:
It showed up like this (the barcode line):
![tracing_log](https://user-images.githubusercontent.com/671465/208347544-c7f72f9c-0b49-473f-95a4-56863f049645.png)

Reply:
Interesting, yeah it does seem useful